spec/spec_tests/uri_options_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'lite_spec_helper'
+require 'spec_helper'
```

Comment:
Changing this because this spec requires `support/cluster_config`.

This causes every test in `uri_options_spec` and `connection_string_spec` to fail with `NoAvailableServer` errors (see [here](https://evergreen.mongodb.com/task_log_raw/mongo_ruby_driver_enterprise_auth_tests_ubuntu__auth_and_ssl~auth_and_ssl_ruby~jruby_9.1_enterprise_auth_tests_patch_310299025045a26be3fa769dd9b45373108968f9_5d5db6e50ae6061643ad02b1_19_08_21_21_25_59/0?type=T&text=true)), only when run from `run-enterprise-auth-tests`. There is probably a gem that I need to require. If you have an idea of what that might be, feel free to let me know. I will investigate more tomorrow.